docs/debugging-s2i.md

```diff
@@ -0,0 +1,49 @@
+Troubleshooting
+=================
+
+This document contains some tips and suggestions for troubleshooting using the source-to-image command ("sti") to inject source into Docker images.
+
+OpenShift Server
+------------------
+
+The "sti" command of course interfaces with an OpenShift v3 server.  The outputted image is uploaded to the server's internal Docker registry.  If your efforts bring into question how the OpenShift server is operating, check out
+some of the tips [here](https://github.com/openshift/origin/blob/master/docs/debugging-openshift.md) for help.
+
+
+Can't Find The "sti" Executable (if you have cloned the repoistory)
+--------------
+
+   After running the build-release.sh script, look under the _output directory.
```

Comment:
I'd suggest just plain old make, there's no need in building a release build, that's the script we use for building the actual release. Or even better, maybe pointing to our releases would be even better.